source/globalCommands.py

```diff
@@ -793,15 +824,40 @@ def script_toggleReportStyle(self, gesture):
 		category=SCRCAT_DOCUMENTFORMATTING,
 	)
 	def script_toggleReportSpellingErrors(self, gesture: inputCore.InputGesture):
```

Comment:
I have not tested this. But reading it, the code seems to be working as expected.

I feel that doing such specific cases with numeric values is not so self explaining.
Why not something like the following:
```
brailleValue = config.conf["documentFormatting"]["reportSpellingErrors2"] & ReportSpellingErrors.BRAILLE
speechAndSoundValue = config.conf["documentFormatting"]["reportSpellingErrors2"] & (ReportSpellingErrors.SPEECH | ReportSpellingErrors.SOUND)
speechAndSoundValue = (speechAndSoundValue + 1) % 4  # Or may we replace 4 by a more self explaining value?
config.conf["documentFormatting"]["reportSpellingErrors2"] = brailleValue | speechAndSoundValue
ui.message(

			_("Report spelling errors {mode}").format(mode=ReportSpellingErrors(speechAndSoundValue).displayString)

		)
```